synapse/third_party_rules/access_rules.py

```diff
@@ -167,8 +171,44 @@ def on_create_room(self, requester, config, is_requester_admin) -> bool:
                 if not allowed:
                     raise SynapseError(400, "Invalid power levels content")
 
+                use_default_power_levels = False
+
+        # If power levels were not overridden by the user, override with DINUM's preferred
+        # defaults instead
+        if use_default_power_levels:
+            config["power_level_content_override"] = self._get_default_power_levels(
+                requester
+            )
+
         return True
 
+    # If power levels are not overridden by the user during room creation, the following
+    # rules are used instead. Changes from Synapse's default power levels are noted.
+    #
+    # The same power levels are currently applied regardless of room preset.
+    @staticmethod
+    def _get_default_power_levels(requester: Requester) -> Dict:
```

Comment:
would it be simpler just to make synapse's default power levels configurable?

Reply:
Possibly, though I could see us wanting to tweak things so that the power levels depend on the room preset, access rules, or some other dependency. Having it be decided by this module that can already hook into that makes the solution seem a lot more future proof.

But yes, it may be nice that have that for other reasons.

Reply:
fair.